pkg/cri/server/container_execsync.go

```diff
@@ -117,6 +117,11 @@ func (c *criService) execInternal(ctx context.Context, container containerd.Cont
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	drainExecSyncIOTimeout, err := time.ParseDuration(c.config.DrainExecSyncIOTimeout)
```

Comment:
if it's nil let's continue with it set to zero duration vs throw an error..

Reply:
I added fallback to zero if it is empty.